src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -605,6 +605,175 @@ def load_cert_validator(namespace):
+            type='storageAccounts',
+            name=namespace.storage_account)
+
+    process_nw_flow_log_show_namespace(namespace)
+
+def process_nw_flow_log_show_namespace(namespace):
+    from azure.mgmt.network import NetworkManagementClient
+    from azure.cli.core.commands.arm import parse_resource_id
+
+    if not is_valid_resource_id(namespace.nsg):
+        namespace.nsg = resource_id(
+            subscription=get_subscription_id(),
+            resource_group=namespace.resource_group_name,
+            namespace='Microsoft.Network',
+            type='networkSecurityGroups',
+            name=namespace.nsg)
+
+    network_client = get_mgmt_service_client(NetworkManagementClient).network_security_groups
+    id_parts = parse_resource_id(namespace.nsg)
+    nsg_name = id_parts['name']
+    rg = id_parts['resource_group']
+    nsg = network_client.get(rg, nsg_name)
+    namespace.location = nsg.location  # pylint: disable=no-member
+    get_network_watcher_from_location(remove=True)(namespace)
+
+
+def process_nw_topology_namespace(namespace):
+
+    location = namespace.location
+    if not location:
+        from azure.mgmt.resource import ResourceManagementClient
+        resource_client = get_mgmt_service_client(ResourceManagementClient).resource_groups
+        resource_group = resource_client.get(namespace.target_resource_group_name)
+        location = resource_group.location  # pylint: disable=no-member
+
+    get_network_watcher_from_location(
+        watcher_name='network_watcher_name', rg_name='resource_group_name')(namespace)
+
+
+def process_nw_packet_capture_create_namespace(namespace):
+
+    get_network_watcher_from_vm(namespace)
+
+    storage_usage = CLIError('usage error: --storage-account NAME_OR_ID [--storage-path '
+                             'PATH] [--file-path PATH] | --file-path PATH')
+    if not namespace.storage_account and not namespace.file_path:
+        raise storage_usage
+
+    if namespace.storage_path and not namespace.storage_account:
+        raise storage_usage
+
+    if not is_valid_resource_id(namespace.vm):
+        namespace.vm = resource_id(
+            subscription=get_subscription_id(),
+            resource_group=namespace.resource_group_name,
+            namespace='Microsoft.Compute',
+            type='virtualMachines',
+            name=namespace.vm)
+
+    if namespace.storage_account and not is_valid_resource_id(namespace.storage_account):
+        namespace.storage_account = resource_id(
+            subscription=get_subscription_id(),
+            resource_group=namespace.resource_group_name,
+            namespace='Microsoft.Storage',
+            type='storageAccounts',
+            name=namespace.storage_account)
+
+    if namespace.file_path:
+        file_path = namespace.file_path
+        if not file_path.endswith('.cap'):
+            raise CLIError("usage error: --file-path PATH must end with the '*.cap' extension")
+        file_path = file_path.replace('/', '\\')
```

Comment:
Update: This works on Windows as well. However, on Windows, you'd likely enter your path using backslashes, but the end result is the same on the server:
```
--file-path /capture/capture.cap   ==> "\\capture\\capture.cap"
--file-path \capture\capture.cap   ==> "\\capture\\capture.cap"
```